### PR TITLE
enh: support websocket stmt query

### PR DESCRIPTION
**Pull request by huskar-t**

### CODE REVIEW

File: taosWS/connection.go [hunk 1]
```diff
@@ -99,9 +108,297 @@ func (tc *taosConn) Close() (err error) {
...
+	tc.buf.Reset()
+	err = jsonI.NewEncoder(tc.buf).Encode(action)
+	if err != nil {
+		return err
+	}
+	err = tc.writeText(tc.buf.Bytes())
+	if err != nil {
+		return err
+	}
+	return nil
```
Comment (sheyanjie-qq) [inline taosWS/connection.go:220]: do we need to check the response?

File: taosWS/statement.go [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+		switch stmt.cols[v.Ordinal-1].FieldType {
+		case common.TSDB_DATA_TYPE_NULL:
+			v.Value = nil
+		case common.TSDB_DATA_TYPE_BOOL:
+			rv := reflect.ValueOf(v.Value)
+			switch rv.Kind() {
+			case reflect.Bool:
+				v.Value = types.TaosBool(rv.Bool())
+			case reflect.Float32, reflect.Float64:
+				v.Value = types.TaosBool(rv.Float() == 1)
```
Comment (sheyanjie-qq) [inline taosWS/statement.go:126]: why not to use > 0?

File: taosWS/statement_test.go [hunk 3]
```diff
@@ -0,0 +1,2159 @@
...
+		t.Error(err)
+		return
+	}
+	stmt, err := db.Prepare("insert into test_stmt_driver_ws_q.ct values (?,?,?,?,?,?,?,?,?,?,?,?,?,?)")
+	if err != nil {
+		t.Error(err)
+		return
+	}
+	now := time.Now()
+	result, err := stmt.Exec(now, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, "binary", "nchar")
```
Comment (sheyanjie-qq) [inline taosWS/statement_test.go:114]: can we insert many rows by call exec once?

File: taosWS/connection.go (hunk 1 above)
Comment (huskar-t) [reply taosWS/connection.go:220]: There will be no response

File: taosWS/statement_test.go (hunk 3 above)
Comment (huskar-t) [reply taosWS/statement_test.go:114]: Only a single SQL statement is allowed

File: taosWS/statement.go (hunk 2 above)
Comment (huskar-t) [reply taosWS/statement.go:126]: "1" represents true in the strconv.ParseBool method. It can also be changed to > 0 for true

**Review by sheyanjie-qq — APPROVED** [review]
LGTM

**Pull request merged**